packages/lit-dev-content/src/components/litdev-playground-page.ts

```diff
@@ -104,13 +141,20 @@ export class LitDevPlaygroundPage extends LitElement {
     };
     this._playgroundProject = mustFindChild('playground-project')!;
     this._playgroundPreview = mustFindChild('playground-preview')!;
+    this._fileEditor = mustFindChild('playground-file-editor')!;
     this._shareButton = mustFindChild('litdev-playground-share-button')!;
     this._downloadButton = mustFindChild('litdev-playground-download-button')!;
     this._examplesDrawer = mustFindChild('litdev-drawer')!;
     this._exampleControls = mustFindChild('litdev-example-controls')!;
     this._examplesDrawerScroller = mustFindChild(
       'litdev-drawer .minimalScroller'
     )!;
+    // Unforuntately, the file editor does not emit a change event, so we reach
+    // into the shadow root and find the code editor that does.
+    await this._fileEditor.updateComplete;
+    this._codeEditor = this._fileEditor.shadowRoot?.querySelector(
+      'playground-code-editor'
+    )!;
```

Comment:
Super minor nit: Since this is getting `!` asserted as it must exist, would it be semantically clearer to also `!` for the `.shadowRoot`?
```suggestion
    this._codeEditor = this._fileEditor.shadowRoot!.querySelector(
      'playground-code-editor'
    )!;
```